manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -0,0 +1,222 @@
+// @ts-ignore
+import { MantaPrivateWallet, Environment, Network, MantaUtilities } from 'manta.js';
+import BN from 'bn.js';
+import { web3Accounts, web3Enable, web3FromSource } from '@polkadot/extension-dapp';
+
+async function main() {
+    await toPrivateOnlySignTest();
+    await toPrivateTest();
+    await privateTransferTest();
+    await toPublicTest();
+    await publicTransferTest();
+    console.log("END");
+}
+
+// Get Polkadot JS Signer and Polkadot JS account address.
+const getPolkadotSignerAndAddress = async () => {
+    const extensions = await web3Enable('Polkadot App');
+    if (extensions.length === 0) {
+        throw new Error("Polkadot browser extension missing. https://polkadot.js.org/extension/");
+    }
+    const allAccounts = await web3Accounts();
+    let account = allAccounts[0];
+
+    const injector = await web3FromSource(account.meta.source);
+    const polkadotSigner = injector.signer;
+    const polkadotAddress = account.address;
+    return {
+        polkadotSigner,
+        polkadotAddress
+    }
+}
+
+/// Test to publicly transfer 10 DOL.
+const publicTransferTest = async () => {
+    const privateWalletConfig = {
+        environment: Environment.Development,
+        network: Network.Dolphin
+    }
+
+    const privateWallet = await MantaPrivateWallet.init(privateWalletConfig);
+    const polkadotConfig = await getPolkadotSignerAndAddress();
+    const mantaUtilities = new MantaUtilities();
```

Comment:
we can remove this instance of mantaUtilities since all methods can be statically called now. replace "mantaUtilities" with "MantaUtilities"